Rules/UseConsistentWhitespace.cs

```diff
@@ -211,6 +235,112 @@ private IEnumerable<DiagnosticRecord> FindOpenBraceViolations(TokenOperations to
             }
         }
 
+        private IEnumerable<DiagnosticRecord> FindInnerBraceViolations(TokenOperations tokenOperations)
+        {
+            foreach (var lCurly in tokenOperations.GetTokenNodes(TokenKind.LCurly))
+            {
+                if (lCurly.Next == null
+                    || !IsPreviousTokenOnSameLine(lCurly)
+                    || lCurly.Next.Value.Kind == TokenKind.LCurly
```

Comment:
does this mean the rule will skip things like `{{{{{`?

Reply:
yes, you are right, I've taken that line out now so that it formats to `{ { { foo } } }`, etc. if there is more than 1 consecutive brace